src/index.html

```diff
@@ -7,7 +7,7 @@
     <title>To Do List</title>
   </head>
   <body>
-    <div class="container">
+    <div class="containe">
       <div class="title-main-container">
         <div class="title-container">
```

Comment:
OPTIONAL
Just use sections instead of divs

Reply:
thanks, I will do it.